system/system_test.go

```diff
@@ -127,6 +133,51 @@ func (tc *fakeTracerouteClient) Recv() (*spb.TracerouteResponse, error) {
 	return resp, tc.err
 }
 
+func TestSwitchControlProcessor(t *testing.T) {
```

Comment:
move this after TestPing, to keep them alphabetical

Reply:
done